ui/src/components/shared/Detail.tsx

```diff
@@ -1,19 +1,31 @@
-import { Row, Typography } from "antd";
+import { Row, Tag, Typography } from "antd";
 
 import { ReactComponent as IconCheckMark } from "src/assets/icons/check.svg";
 import { ReactComponent as IconCopy } from "src/assets/icons/copy-01.svg";
 
 export function Detail({
   copyable,
   ellipsisPosition,
+  flavor = {
+    type: "plain",
+  },
   label,
   text,
 }: {
   copyable?: boolean;
   ellipsisPosition?: number;
+  flavor?:
+    | {
+        type: "plain";
+      }
+    | {
+        color?: string;
+        type: "tag";
+      };
```

Comment:
Instead of flavor just tag
```suggestion
  tag?: PresetStatusColorType
```

Reply:
What is `PresetStatusColorType`?

When you propose `tag`, I assume you are not referring to a UX "tag" but to a kind of `_tag` discriminator, right?

Reply:
Done as agreed. Now instead of `flavor` we can optionally pass a `TagProps` object to render the detail as a tag.